tests/tools/dumping/test_processes.py

```diff
@@ -266,14 +275,31 @@ def test_dump_calculation_flat(tmp_path, generate_calculation_node_io):
 
 
 # Here, in principle, test only non-default arguments, as defaults tested above
-# @pytest.mark.parametrize('overwrite', (True, False))
-def test_dump_calculation_overwrite(tmp_path, generate_calculation_node_io):
+def test_dump_calculation_overwr_incr(tmp_path, generate_calculation_node_io):
```

Comment:
```suggestion
def test_dump_calculation_overwr_incr(tmp_path, generate_calculation_node_io):
    """Tests the ProcessDumper for the overwrite and incremental option."""
```
Maybe also add tests for `incremental=True`?

Reply:
Ah sorry, it was hidden by GitHub. I think the case `overwrite=True, incremental=True`  is missing, but if you don't want to implement it, I am okay. code coverage is positive already